### ISSUE TITLE
It's not work

### ISSUE THREAD

**Issue by Here21** [qnub/meteor-accounts-react-material-ui#2]
I really like your this component, but I still inexperienced, but depending on your method I use this component, it does not appear on the page.I hope you can publish a detailed usage

**Comment by qnub** [comment]
Try to add `<Accounts.ui.LoginFormSet/>` to your component which already displayed on page.

Example:

```
MyComponent = React.createClass({
    render(){
         return(<div>
            <h1>Hello world</h1>
            <Accounts.ui.LoginFormSet/>
        </div>);
    }
});
```

Of course you should have fully working [React Material UI](http://material-ui.com/#/home) in your project… Like configured browserify, injected TouchTap event and so on. You can try [some existed packages](https://atmospherejs.com/?q=material%20ui)

**Comment by Here21** [comment]
I already install  izzilab:material-ui and qnub:accounts-react-material-ui，but nothing on page
[this is demo source](https://github.com/Martin-zhz1994/Projects-Meteor-MakerLab/blob/fa078435d45b6dbd755b316aac112d13a59b6616/client/Login.jsx)

**Comment by qnub** [comment]
Ok, i've nailed your problem. You should have installed at leas one accounts-service like `accounts-password`, `accounts-facebook` or any other:

```
meteor add accounts-password
```

![image](https://cloud.githubusercontent.com/assets/496270/11390317/c1bce200-936a-11e5-83c7-db3740fe9443.png)

I've released new version with 0.13.4 material ui and updated README.

**Comment by Here21** [comment]
Awesome！Thank you very much！！！
